flutter_local_notifications/android/src/main/java/com/dexterous/flutterlocalnotifications/FlutterLocalNotificationsPlugin.java

```diff
@@ -1224,4 +1230,26 @@ private void getActiveNotifications(Result result) {
             result.error(GET_ACTIVE_NOTIFICATIONS_ERROR_CODE, e.getMessage(), e.getStackTrace());
         }
     }
+    private void getNotificationChannels(Result result) {
+        if (VERSION.SDK_INT < VERSION_CODES.O) {
+            result.error(GET_NOTIFICATION_CHANNELS_ERROR_CODE, GET_NOTIFICATION_CHANNELS_ERROR_MESSAGE, null);
+            return;
+        }
+        try {
+            NotificationManager notificationManager = (NotificationManager) applicationContext.getSystemService(Context.NOTIFICATION_SERVICE);
+            List<NotificationChannel>  channels = notificationManager.getNotificationChannels();
+            List<Map<String, Object>> channelsPayload = new ArrayList<>();
+
+            for (NotificationChannel channel : channels) {
+                HashMap<String, Object> channelPayload = new HashMap<>();
```

Comment:
Made a comment on this in the `notification_channel_output.dart` file but there's other info that could be retrieved to populate the remaining properties of the `AndroidNotificationChannel` Dart class that already exists and would remove the need to have the new class that was created in this PR. Would you be able to make changes in this PR to do this?